### DISCUSSION TITLE
[Unstable] Flare and glowstick activation method is inconsistent with grenades

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14591] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

you use left mouse to lit the flare/glowstick
you use E to pull the pin

### Reproduction steps

1. spawn flare or glowstick and try to lit then
2. spawn grenade and try to pull the pin

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by lIllIIlI** [comment]
>you use E to pull the pin

huh? that's a thing???

**Reply by A2ur3** [reply]
unstable changes

**Comment by ubertpendragon** [comment]
I agree but **the main issue is that you can't have a weapon in 1 hand and a flare in the other** since it will always throw the flare

Maybe make flares not throwable and keep glowsticks throwable?

**Comment by lIllIIlI** [comment]
either of the two would work:
make flare/glowsticks attachable to your character
or rightclick + e to swap the throwing mode for the throwables in hand, i.e in one mode the flare will be held like a riotshield is (no left click action) and the other mode could be throw on click.

**Reply by jigardi** [reply]
This should really be a base game functionality, it would add a considerable amount of fun to the game:

"Oi! Could you throw me that welder over there?" "Sure thing boss!" *ThrowsWelderToBoss*

"Aaaagh! I'm out of bullets!" "We're all out of bullets!" "What do I do?!" "THROW THE GUN! THROW THE GUN AT IT!"
